src/lib/intermediate-config-collector-builder.ts

```diff
@@ -0,0 +1,37 @@
+import { sortBy } from 'lodash';
+import {LibraryConfig} from "./interfaces/library-config";
+import {IntermediateConfigCollector, IntermediateExtendsEntry} from "./interfaces/intermediate-config-collector";
+
+const getEntriesForLibConfig = (libConfig: LibraryConfig): IntermediateExtendsEntry[] =>
+    libConfig.extendsConfigs.map(config => ({
+        name: config,
+        order: libConfig.order||'normal'
```

Comment:
It would make sense to extract it to a constant like `const DEFAULT_ORDER = 'normal';`, but still good.